tutorials/0_to_litgpt.md

```diff
@@ -0,0 +1,549 @@
+# Zero to Pretraining, Finetuning, and Using LLMs with LitGPT
+
+
+
+This tutorial walks you through the main features and usage patterns for ⚡️LitGPT, a library for pretraining, finetuning, and using LLMs that focuses on an efficient user experience while being developer-friendly.
+
+The topics, following the installation of LitGPT, are in chronological order, reflecting the steps in an LLM lifecycle: Pretraining → Finetuning → Inference.
+
+&nbsp;
+
+<img src="images/0_to_litgpt/usage.webp" width=500>
+
+&nbsp;
+
+<img src="images/0_to_litgpt/4-commands.webp" width=300>
+
+&nbsp;
+
+However, it is also possible, and even common, to use and deploy models with LitGPT without pretraining and finetuning. So, if you are not interested in pretraining and finetuning, please feel free to skip these sections.
+
+
+
+
+
+&nbsp;
+## Install LitGPT
+
+LitGPT is available as a Python library from the PyPI package repository, and we recommend installing it using Python's `pip` installer module, including all required package dependencies:
+
+```bash
+pip install 'litgpt[all]' 
+```
+
+Alternatively, if you are a researcher or developer planning to make changes to LitGPT, you can clone the GitHub repository and install it from a local folder as follows:
+
+```
+git clone https://github.com/Lightning-AI/litgpt.git
+cd litgpt
+pip install -e '.[all]'
+```
```

Comment:
Is it worth repeating the installation instructions again? Wouldn't the viewer have arrived to this tutorial from the README?

Reply:
> I don't love the "zero to litgpt" title

Not my choice :P. Let's discuss "Getting Started" 

> Is it worth repeating the installation instructions again? Wouldn't the viewer have arrived to this tutorial from the README?

Yes, I think this is valuable because this way we can link the document as a standalone resource.